src/gsy_e/gsy_e_core/user_profile_handler.py

```diff
@@ -151,6 +152,18 @@ def _buffer_profile_uuid_list(self):
 
         self._profile_uuids = list(profile_selection)
 
+    @db_session
+    def _buffer_profile_types(self):
+        """ Buffers list of the profile_uuids that correspond to this simulation into
+        self._profile_uuids"""
```

Comment:
Nit (but since it's in the PEP and in our style conventions):
```suggestion
        """
        Buffer a list of the profile_uuids that correspond to this simulation into
        self._profile_uuids.
        """
```

Reply:
I'll also fix the docstring, thanks